services/management/adapter/memory_provider.go

```diff
@@ -66,6 +68,20 @@ func getCommitteeFromConfig(config MemoryConfig) []primitives.NodeAddress {
 	return committee
 }
 
+func getTopologyFromConfig(cfg MemoryConfig) []*services.GossipPeer {
+	peers := cfg.GossipPeers()
+	topology := make([]*services.GossipPeer, 0, len(peers))
+	for _, peer := range peers {
+		if nodeAddress, err := hex.DecodeString(peer.HexOrbsAddress()); err != nil {
```

Comment:
reverse the condition and throw away empty clause?

Reply:
added log and comment